tsconfig.json

```diff
@@ -11,46 +11,37 @@
     },
     "types": ["node"],
     "exclude": [
-        "./data/mods/gen1/pokedex.js"
+        "./data/mods/gen1/pokedex.js",
+        "./server/index.js",
+        "./server/chat-commands.js"
     ],
     "include": [
         "./config/formats.js",
         "./data/*",
+        "./data/mods/*/*",
         "./dev-tools/global.d.ts",
         "./dev-tools/globals.ts",
         "./dev-tools/node.d.ts",
-        "./sim/*.js",
         "./lib/*.js",
-        "./server/dnsbl.js",
-        "./server/ladders-local.js",
-        "./server/ladders-remote.js",
-        "./server/ladders.js",
-        "./data/mods/*/*",
-        "./server/monitor.js",
-        "./server/loginserver.js",
-        "./server/rooms.js",
-        "./server/roomlogs.js",
-        "./server/chat-formatter.js",
-        "./server/room-game.js",
-        "./server/room-battle.js",
-        "./server/team-validator-async.js",
-        "./server/verifier.js",
-        "./server/chat.js",
-        "./server/users.js",
+        "./server/*.js",
         "./server/chat-plugins/chat-monitor.js",
         "./server/chat-plugins/daily-spotlight.js",
         "./server/chat-plugins/hangman.js",
         "./server/chat-plugins/helptickets.js",
-        "./server/chat-plugins/mafia.js",
         "./server/chat-plugins/mafia-data.js",
+        "./server/chat-plugins/mafia.js",
         "./server/chat-plugins/othermetas.js",
         "./server/chat-plugins/poll.js",
         "./server/chat-plugins/room-faqs.js",
         "./server/chat-plugins/thecafe.js",
         "./server/chat-plugins/thing-of-the-day.js",
         "./server/chat-plugins/uno.js",
         "./server/chat-plugins/wifi.js",
-        "./server/punishments.js",
-        "./server/tournaments/generator-round-robin.js"
+        "./server/dnsbl.js",
+        "./server/ladders-local.js",
+        "./server/ladders-remote.js",
+        "./server/ladders.js",
```

Comment:
These four seem unnecessary with your `./server/*.js` matcher. Is the `.js` part necessary though?

Reply:
oops, these were hiding. 

I had kept the `.js` for consistency with the `sim/` and `lib/`, but i guess you did it there because you didn't want Typescript to look at the `.md` files (why care? it doesn't do anything with them, and I guarantee this isn't what its making `tsc` slow)

Reply:
Oh, I think I didn't know tsc ignores `.md` files instead of breaking on them. I'd support changing them all to `*` if that's not a problem for `tsc`.

Reply:
I verified it isn't, I'll update.